### ISSUE TITLE
fastboot support

### ISSUE THREAD

**Issue by st-h** [kaliber5/ember-stargate#64]
Is this addon currently supposed to work in fastboot mode?

From my observations, that does not seem to be the case:

```
This is rendered in fastboot

<Portal @target="modal">
  This does not show up in the DOM in fastboot mode
</Portal>
```

Everything works fine in the browser, though 👍

**Comment by simonihmig** [comment]
Yeah, you are right, this does not (currently) work in FastBoot. Need to think if/how this would be possible with the current API...

**Comment by st-h** [comment]
> Yeah, you are right, this does not (currently) work in FastBoot. Need to think if/how this would be possible with the current API...

Ah, I think I can see why this is happening. We are having similar issues with using the `did-insert` modifier in our app. Probably having a modifier similar to render-modifiers which provides fastboot support as well, would be a good start here?
I just asked in fastboot-discord, as I am unsure if this actually would be a good way to progress 🤷‍♀️
